src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -642,17 +642,24 @@ public static double getPlayerVisibilityDistance(EntityPlayer player, double xzD
         return value >= maxXZDistance ? maxXZDistance : value;
     }
 
+    @Deprecated // Kept here to avoid binary compatability break
     public static boolean isLivingOnLadder(@Nonnull IBlockState state, @Nonnull World world, @Nonnull BlockPos pos, @Nonnull EntityLivingBase entity)
+    {
+        return isLivingOnLadder(state, world, pos, entity, null);
+    }
+    
+    public static boolean isLivingOnLadder(@Nonnull IBlockState state, @Nonnull World world, @Nonnull BlockPos pos, @Nonnull EntityLivingBase entity, EnumFacing side)
     {
         boolean isSpectator = (entity instanceof EntityPlayer && ((EntityPlayer)entity).isSpectator());
         if (isSpectator) return false;
+        if (side == null) return false;
```

Comment:
Might eventually need to run old behaviour if side is null, cause people might expect that to happen... 
Though I don't think it is that commonly used as that one would really need to keep behaviour at that level... 
What do you guys think?